src/components/calculator.js

```diff
@@ -42,4 +42,4 @@ const Calculator = () => {
   );
 };
 
```

Comment:
- [ ] [REQUIRED] Good job so far! You have used `arrow function` method in this file which is very good 👍 but in this milestone you should use `class-based` method looks like that =>

```
// ./src/components/Calculator.js

import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator; // export Calculator component

```

![image](https://user-images.githubusercontent.com/63915024/199792661-7a68212a-f11a-490e-8d9e-e3d9728b6ae7.png)